api/v1beta1/backup_types.go

```diff
@@ -81,9 +81,9 @@ type BackupStatus struct {
 	// Message on the last operation
 	// +kubebuilder:validation:Optional
 	LastMessage string `json:"lastMessage"`
-	// Velero operation status
+	// Velero Backups operation status
 	// +kubebuilder:validation:Optional
-	VeleroBackup *veleroapi.Backup `json:"veleroBackup,omitempty"`
+	VeleroBackups []*veleroapi.Backup `json:"veleroBackup,omitempty"`
```

Comment:
fixed s; 
The reason I went with an array instead of separate resources was that I wasn't sure if we need to add more backup files to the list. It was easier to work with a generic list then to create 3 types of resources